src/pytest_html/scripts/main.js

```diff
@@ -131,10 +111,11 @@ const renderPage = () => {
     const filteredTests = manager.testSubset
     const allTests = manager.allTests
     const collectedItems = manager.collectedItems
+    const runningState = manager.runningState
 
```

Comment:
this could be written cleaner using a destruct:
`const { filteredTests, allTests, collectedItems, runningState } = manager`